core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -106,56 +107,71 @@ class BpmBillingProjectLifecycle(
         )
       })
 
+    def addMembersToBillingProfile(profileModel: ProfileModel): Future[Set[Unit]] = {
+      val members = createProjectRequest.members.getOrElse(Set.empty)
+      Future.traverse(members) { member =>
+        Future(blocking {
+          billingProfileManagerDAO.addProfilePolicyMember(profileModel.getId,
+                                                          ProfilePolicy.fromProjectRole(member.role),
+                                                          member.email,
+                                                          ctx
+          )
+        })
+      }
+    }
+
     createBillingProfile.flatMap { profileModel =>
-      createLandingZone(profileModel)
-        .flatMap { landingZone =>
-          (for {
-            _ <- Option(landingZone.getErrorReport).traverse { errorReport =>
-              Future.failed(
-                new LandingZoneCreationException(
-                  RawlsErrorReport(StatusCode.int2StatusCode(errorReport.getStatusCode), errorReport.getMessage)
+      addMembersToBillingProfile(profileModel).flatMap { _ =>
```

Comment:
do I also need to add the users to the Landing Zone here? I have essentially zero knowledge of how that stuff works

Reply:
I don't believe you need to worry about the landing zone, but I will check out the branch tomorrow and give it a try.

Reply:
I was able to create an Azure billing project, including 2 users at billing project creation time (existing user with role user, new user with role owner). They were both able to view and use the billing project to create a workspace.

I did not get the invitation email, but that's to be expected because my local notification configuration did not support it (notifications.topicName=workbench-notifications-local… that needs to read notifications.topicName=workbench-notifications-dev  for the email to go through)